Sources/Baggage/BaggageContext.swift

```diff
@@ -98,17 +90,30 @@ public protocol BaggageContextProtocol {
     ///     }
     subscript<Key: BaggageContextKey>(_ key: Key.Type) -> Key.Value? { get set }
 
-    /// Iterates over the baggage context's contents invoking the callback one-by one.
+    /// Calls the given closure on each key/value pair in the `BaggageContext`.
     ///
-    /// - Parameter callback: invoked with the type erased key and value stored for the key in this baggage.
-    func forEach(_ callback: (AnyBaggageContextKey, Any) -> Void)
+    /// - Parameter body: A closure invoked with the type erased key and value stored for the key in this baggage.
+    func forEach(_ body: (AnyBaggageContextKey, Any) throws -> Void) rethrows
```

Comment:
New naming and slightly tweaked docs inspired by [`Sequence.forEach`](https://github.com/apple/swift/blob/master/stdlib/public/core/Sequence.swift#L684).